docs/troubleshooting.adoc

```diff
@@ -0,0 +1,157 @@
+= Troubleshooting Landrush
+:toc:
+:toc-placement!:
+
+Use this document to identify and resolve basic Landrush issues that you encounter.
+https://github.com/vagrant-landrush/landrush#landrush-dns-for-vagrant-[Landrush]
+is a platform-agnostic DNS server used by ADB to provide name based access to
+services, such as Openshift, running in the ADB.
+
+As an ADB user, you download Landrush as a vagrant plugin. For further details
+on downloading Landrush, refer to the
+link:installing.rst[Installation document].
+
+'''
+toc::[]
+'''
+
+==== I am unable to access the Internet or override the Landrush upstream DNS?
+By default, Landrush uses Google's DNS server, with the IP address 8.8.8.8, to
+resolve DNS queries that are not for Landrush managed hostnames. If your network
+blocks access to the Google DNS servers, due to firewalls, VPN and so on, as a workaround,
+you need to set a custom upstream DNS server. This server is configured with the
+`config.landrush.upstream` option.
+
+For example, to use 10.25.35.4 as a DNS server, add this line to your `Vagrantfile`:
+
+`config.landrush.upstream '10.25.35.4'`
+
+You will need to check your network configuration on the host or ask your network
+administrator for the right DNS server address to use.
+
+For further details, refer to the
+https://github.com/vagrant-landrush/landrush#guest-is-unable-to-access-the-internet[Guest is unable to access the Internet]
+and the https://github.com/vagrant-landrush/landrush#unmatched-queries[Unmatched Queries]
+sections in the Landrush documentation.
+
+For future reference you can track this https://github.com/projectatomic/adb-atomic-developer-bundle/issues/523[issue].
+
+==== I cannot access OpenShift console from my browser.
+Landrush enables you to access services running in the ADB,  such as OpenShift,
```

Comment:
Currently, all the information here has been drawn from: https://github.com/projectatomic/adb-atomic-developer-bundle/issues/525. If you could add more information in that issue, or here itself, I could work on it further. 
If not, it would be great if you could open a new doc issue here once you are done with the documentation task in Landrush.